man/nextcloudcmd.1.rst

```diff
@@ -16,7 +16,7 @@ Contrary to the :manpage:`nextcloud(1)` GUI client, `nextcloudcmd` only performs
 a single sync run and then exits. In so doing, `nextcloudcmd` replaces the
 `ocsync` binary used for the same purpose in earlier releases.
 
-A *sync run* synchronizes a single local directory using a WebDAV share on a
+A *sync run* synchronizes a single local directory using a DAV share on a
```

Comment:
is DAV really correct? Should it not be WebDAV? the protocol is still the WebDAV protocol. It's just that our endpoint changed.

Reply:
@FlexW true
the protocol is really WebDAV but the endpoint of the server is going to be deprecated and seeing all our difficulties I understand that the clean up can maybe remove quite some code from them and we have already removed some code from our own code